frontend/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-container fluid>
+    <content-card
+      class="contentCard"
+      :title="$tc('views.pageNotFound.title')"
+      max-width="1200"
+      toolbar
+    >
+      <template #title-actions>
+        <v-btn
+          class="d-md-none"
+          icon
+          :to="{ name: 'profile', query: { isDetail: true } }"
+        >
+          <user-avatar :user="user" :size="36" />
+        </v-btn>
+      </template>
+
+      <v-card-text class="justify-center">
+        <v-img contain class="error-image" src="@/assets/FourZeroFour.svg"></v-img>
```

Comment:
using v-img

Reply:
No, please just use img, as explained above. It does not get displayed for me with v-img. It should not be a background image.